### PR TITLE
Add app service sample entries

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: azure-samples/samples.json [hunk 1]
```diff
@@ -414,6 +420,18 @@
       "githubPath": "Azure-Samples/app-service-java-manage-storage-connections-for-web-apps",
       "description": "Getting started on managing storage connections for Web Apps using Java"
     },
+    {
+      "owner": "jianghaolu",
+      "filePath": "azure-samples/src/main/java/com/microsoft/azure/management/appservice/samples/ManageWebAppCosmosDbByMsi.java",
+      "githubPath": "Azure-Samples/app-service-java-access-key-vault-by-msi-for-web-apps",
+      "description": "Getting started on accessing Key Vault with Managed Service Identity for Web Apps using Java"
```
Comment (selvasingh) [inline azure-samples/samples.json]: Suggest a different description - Getting started on safeguarding Web app secrets in Key Vault

File: azure-samples/samples.json [hunk 2]
```diff
@@ -414,6 +420,18 @@
...
+    },
+    {
+      "owner": "jianghaolu",
+      "filePath": "azure-samples/src/main/java/com/microsoft/azure/management/appservice/samples/ManageWebAppCosmosDbByMsi.java",
```
Comment (selvasingh) [inline azure-samples/samples.json]: This should point to a different Java file - ManageWebAppCosmosDbThroughKeyVault.java

File: azure-samples/samples.json [hunk 3]
```diff
@@ -414,6 +420,18 @@
...
+      "githubPath": "Azure-Samples/app-service-java-access-key-vault-convenience-for-web-apps",
+      "description": "Getting started on accessing Key Vault through the convenience layer with Managed Service Identity for Web Apps using Java"
```
Comment (selvasingh) [inline azure-samples/samples.json]: Suggest a different description - Getting started on safeguarding Web app secrets in Key Vault using convenience API

**Comment by anuchandy** [comment]
closing this PR, the comments are addressed in the PR https://github.com/Azure/azure-libraries-for-java/pull/166

**Pull request closed without merging**